core/drivers/clk/clk.c

```diff
@@ -540,9 +540,8 @@ static void print_tree(void)
 
 void clk_print_tree(void)
 {
-	if (IS_ENABLED(CFG_DRIVERS_CLK_PRINT_TREE) &&
-	    TRACE_LEVEL >= TRACE_DEBUG) {
-		DMSG("Clock tree summary (informative):");
+	if (IS_ENABLED(CFG_DRIVERS_CLK_PRINT_TREE)) {
```

Comment:
Removing test on `TRACE_LEVEL >= TRACE_INFO` makes that when `CFG_DRIVERS_CLK_PRINT_TREE` is enabled but `TRACE_LEVEL` to low, there will be useless code embedded.
I would be in favor of restoring this test.

Ditto in `core/drivers/regulator/regulator.c`.

Reply:
done, thank you!